modules/social_features/social_comment/social_comment.tokens.inc

```diff
@@ -212,7 +212,7 @@ function social_comment_tokens($type, $tokens, array $data, array $options, Bubb
                   }
                   else {
                     if ($entity instanceof NodeInterface) {
-                      $commented_content_type = strtolower($entity->type->entity->label());
+                      $commented_content_type = strtolower($entity->getType());
```

Comment:
```
 /**
   * Gets the node type.
   *
   * @return string
   *   The node type.
   */
  public function getType();
```

NodeInterface::getType(). This is returning label.

Reply:
```
 /**
   * {@inheritdoc}
   */
  public function getType() {
    return $this->bundle();
  }
```

Reply:
@ronaldtebrake Do you mean that this is returning Bundle's label? This what I found in ContentEntityBase

```
 /**
   * {@inheritdoc}
   */
  public function bundle() {
    return $this->getEntityKey('bundle');
  }
```

Reply:
Correct! 
The key is likely a machine name of the actual entity, instead of the label that was given when creating it.
So imagine your content type is created as Super Duper Content the bundle will likely be called super_duper_content. (Didn't test this, it's just an assumption)

Reply:
I updated and replaced with `$entity->getEntityType()->getLabel()`